wled/wled.py

```diff
@@ -344,6 +344,13 @@ async def playlist(self, playlist: int) -> None:
         """Set a running playlist on a WLED device."""
         await self._request("state", method="POST", json_data={"pl": playlist})
 
+    async def raw(self, json_data: dict) -> None:
+        """Post a raw state request to WLED.
+
+        See https://github.com/Aircoookie/WLED/wiki/JSON-API
+        """
+        await self._request("state", method="POST", json_data=json_data)
```

Comment:
This is a proxy method, why would we add that?

Reply:
To be able to create a similar service like: https://www.home-assistant.io/docs/mqtt/service/
In the example above if we implement a method for preset cycling now it will probably be changed in the near future again as soon as the playlist API will be fully implemented.

Reply:
Ok, it should add methods for that, not a workaround with a proxy method. I have no interest in adding a method like this.

Nevertheless, thanks for willing to contribute.